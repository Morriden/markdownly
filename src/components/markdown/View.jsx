import React from 'react';
import Editor from './Editor';
import Preview from './Preview';
import styles from './View.css';
import MarkdownList from '../markdownList/MarkdownList';
import Controls from '../buttons/Controls';
import Search from '../search/Search';

const View = () => {

  return (
    <>
      <Controls />
      <MarkdownList />
      <Search />
      <div className={styles.View}>
        <Editor />
        <Preview />
      </div>
    </>   
  );
};

export default View;
