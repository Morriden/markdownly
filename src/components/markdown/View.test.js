import React from 'react';
import { shallow } from 'enzyme';
import View from './View';
import AppProvider from '../../provider/AppProvider';

describe('View component', () => {
  it('renders View', () => {
    const wrapper = shallow(<AppProvider><View /></AppProvider>);
    expect(wrapper).toMatchSnapshot();
  });
});
