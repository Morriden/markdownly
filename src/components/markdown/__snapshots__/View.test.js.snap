// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View component renders View 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "state": Object {
        "currentMarkdownID": 1234,
        "markdownList": Array [
          Object {
            "body": "Look at that markdown file.",
            "id": 1234,
            "title": "whatever",
          },
          Object {
            "body": "Look at that markdown file.",
            "id": 5678,
            "title": "new tab",
          },
        ],
      },
    }
  }
>
  <View />
</ContextProvider>
`;
